hadoop-common-project/hadoop-common/src/main/native/src/org/apache/hadoop/io/erasurecode/erasure_coder.c

```diff
@@ -221,6 +221,5 @@ int generateDecodeMatrix(IsalDecoder* pCoder) {
       }
     }
   }
-
```

Comment:
not needed

Reply:
@steveloughran The purpose of this change is to trigger "precommit-run Centos 7". In https://github.com/apache/hadoop/pull/6987, "precommit-run Centos 7" was not triggered, so can not find this failure.
Now that precommit-run Centos 7 is successful, I will delete this line of code.